jwst/resample/resample_spec_step.py

```diff
@@ -45,6 +46,9 @@ def process(self, input):
         if isinstance(input_models[0], MultiSlitModel):
             # result is a MultiProductModel
             result = self._process_multislit(input_models)
+        elif is_tso(input_models[0]):
```

Comment:
Same comment here: you're equating an observation mode with a data model type, which isn't necessarily consistent. The multi-integration version of a MIR_LRS-FIXEDSLIT exposure will not be TSO, but will be a `CubeModel`. It's the `CubeModel` that can't be handled by `resample_spec`.